recipes/swigcsm/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "csmapi" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/USGS-Astrogeology/swigcsm/archive/{{ version }}.tar.gz
+  sha256: 81a6914facb91cf4d5e7156b59eb37d75c598b13d8de467852e43dac69a6bc58
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.10
+    - swig
+    - python >=3.5
```

Comment:
The ` >=3.5` part can be removed (from both sections), and add `py2k` added to the skip comment above, like `skip: True  # [py2k or win]`

Reply:
:+1: Awesome. Thank you. I am going to open an issue and fix this tonight (Pacific Time Zone).